jerry-core/api/jerry.c

```
@@ -5110,6 +5110,122 @@ jerry_get_container_type (const jerry_value_t value) /**< the container object *
+            jerry_release_value (result);
+            result = jerry_create_boolean (true);
+          }
+          break;
+        }
+#endif /* ENABLED (JERRY_BUILTIN_MAP) || ENABLED (JERRY_BUILTIN_WEAKMAP) */
+#if ENABLED (JERRY_BUILTIN_SET) || ENABLED (JERRY_BUILTIN_WEAKSET) \
+        || ENABLED (JERRY_BUILTIN_MAP) || ENABLED (JERRY_BUILTIN_WEAKMAP)
+        case JERRY_CONTAINER_OP_HAS:
+        {
+          result = jerry_return (ecma_op_container_has (container_object_p, arguments[0], type));
+          break;
+        }
+        case JERRY_CONTAINER_OP_DELETE:
+        {
+          if (type == LIT_MAGIC_STRING_WEAKMAP_UL || type  == LIT_MAGIC_STRING_WEAKSET_UL)
+          {
+            result = jerry_return (ecma_op_container_delete_weak (container_object_p, arguments[0], type));
+            break;
+          }
+          result = jerry_return (ecma_op_container_delete (container_object_p, arguments[0], type));
+          break;
+        }
+        case JERRY_CONTAINER_OP_SIZE:
+        {
+          result = jerry_return (ecma_op_container_size (container_object_p));
+          break;
+        }
+        case JERRY_CONTAINER_OP_CLEAR:
+        {
+          result = jerry_return (ecma_op_container_clear (container_object_p));
+          break;
+        }
+#endif
+        default:
+        {
+          result = jerry_throw (ecma_raise_type_error (ECMA_ERR_MSG ("Unsupported container operation")));
+        }
+      }
+      return result;
+    }
+#endif /* ENABLED (JERRY_BUILTIN_CONTAINER) */
+  }
+  else
+  {
+    JERRY_UNUSED (operation);
+    JERRY_UNUSED (container);
+    JERRY_UNUSED (arguments);
+    JERRY_UNUSED (arguments_number);
+    return jerry_throw (ecma_raise_type_error (ECMA_ERR_MSG ("Container is not a object.")));;
+  }
+  JERRY_UNUSED (operation);
+  JERRY_UNUSED (container);
+  JERRY_UNUSED (arguments);
+  JERRY_UNUSED (arguments_number);
+  return jerry_throw (ecma_raise_type_error (ECMA_ERR_MSG ("Containers are disabled.")));;
```

I would throw this error if containers are disabled regardless of input.